man/fi_verbs.7.md

```diff
@@ -177,8 +177,10 @@ The verbs provider checks for the following environment variables.
   deprecated) EP type supports only 1
 
 *FI_VERBS_INLINE_SIZE*
-: Default maximum inline size. Actual inject size returned in fi_info
-  may be greater (default: 64)
+: Maximum inline size for the verbs device. Actual inline size returned may be
+  lower if not supported by the device. This value will be returned by fi_info
```

Comment:
Actual inline size can also be higher than the value. For example, on success, qp_attr.cap.max_inline_data can be greater than the input value.

Reply:
could change to `Actual inline size returned may be different depending on the device capability`.